.github/workflows/ci.yml

```diff
@@ -510,6 +528,80 @@ jobs:
               body: 'Failed to pass the component examples UI Test. If this PR is for UI change and the error is snapshot mismatch, please add "ui change" label to the PR for updating the snapshot.'
             })
 
+  build_upload_base_bundle_stats:
```

Comment:
Just curious if we can store the base stats in our repo and compare to the current stats to those stats instead of having to build each sample on every CI run. Then when merging, we just replace the base stats with the current stats.  But we do need to ensure that the base stats are protected from modification. Just concerned since the obtaining the base bundle stats takes a total of 11 min per CI run.
![image](https://user-images.githubusercontent.com/79475487/167511685-a4d0d757-a67f-4f9d-8841-3e1d76e71107.png)

Reply:
That was what I started with, but those json file take 13mb for each to store, 3x13MB will be around 60MB, which doesn't make a lot of senses to store such a big file 

BTW, building the base will be always faster than other CI tasks since they are running in parallel - building base will only take time to build the base, but all the other CI needs do more steps, so this wouldn't be a bottleneck here

Reply:
I would also like to discourage us from build base each time just to get stats diffs.

Let's remove the base builds from this PR before merging. We'll still get the per-PR size artifacts that we can compare against PRs. We can then figure out how to track the diff.